pkg/api/apis/operators/v1alpha1/clusterserviceversion_types.go

```diff
@@ -278,11 +287,14 @@ type ClusterServiceVersionStatus struct {
 	Conditions []ClusterServiceVersionCondition `json:"conditions,omitempty"`
 	// The status of each requirement for this CSV
 	RequirementStatus []RequirementStatus `json:"requirementStatus,omitempty"`
+	// Time to refresh generated owned APIService certs
+	// +optional
+	CertRefresh metav1.Time `json:"certRefresh,omitempty"`
```

Comment:
Does it make more sense for the status to store `CertLastUpdated` and `CertExpiresAt`?

Reply:
Yes, it does. I just didn't want to hide when the cert was actually expected to be refreshed: `CertRefresh = ExpiresAt - MinFresh`.